### ISSUE TITLE
extraArgs need to be itemize

### ISSUE THREAD

**Issue by zethis** [oauth2-proxy/manifests#101]
extraArgs seems to be mandatory to be used with google provider.

It would be cool to have all the extraArgs details in the documentation:

Here is my configuration to have it working with k8s:

```
        - --cookie-domain=.domain2.com
        - --cookie-domain=.domain1.com
        - --cookie-secure=false
        - --email-domain=*
        - --provider=google
        - --whitelist-domain=.domain2.com
        - --whitelist-domain=.domain1.com
```

Plus:
I'm not able to have multi whitelist-domain entry with helm.
I'm not able to have multi hosts in ingress.

**Comment by pierluigilenoci** [comment]
Feel free to open a PR.

**Comment by kladiv** [comment]
+1
oauth2-proxy helm chart seems not accept multiple cookie-domain and multiple whitelist-domain

**Comment by zethis** [comment]
> +1 oauth2-proxy helm chart seems not accept multiple cookie-domain and multiple whitelist-domain

You need to use config.configFile to do that.

Here an example:

```yaml
config:
  configFile: |-
    email_domains = [ "*" ]
    upstreams = [ "file:///dev/null" ]
    cookie_secure = "false"
    cookie_domains = [ ".domain.com", ".otherdomain.io" ]
    whitelist_domains = [ ".domain.com", ".otherdomain.io"]
    provider = "google"
```

**Comment by pierluigilenoci** [comment]
@zethis would you like to do a PR to integrate documentation?

**Comment by pierluigilenoci** [comment]
@zethis any update on this?

**Comment by pierluigilenoci** [comment]
Closed with #114
